src/wizard/obtain_source.jl

```diff
@@ -168,6 +168,8 @@ function download_source(state::WizardState)
             source_path = joinpath(state.workspace, "$(name)_$n$ext")
         end
 
+        # Initialise `gen_download_cmd`
+        Pkg.probe_platform_engines!()
         download_cmd = Pkg.PlatformEngines.gen_download_cmd(url, source_path)
```

Comment:
`gen_download_cmd` was crashing badly without a call to `Pkg.probe_platform_engines!()`, did anything change in Pkg?

Can we just use `Downloads.download` instead?

Reply:
Yes, we should use that everywhere.